Vagrantfile

```diff
@@ -24,8 +24,8 @@ echo 'export GOSRC=$GOPATH/src' >> /etc/profile.d/envvar.sh
 echo 'export PATH=$PATH:/usr/local/go/bin:$GOBIN' >> /etc/profile.d/envvar.sh
 echo "export http_proxy='$4'" >> /etc/profile.d/envvar.sh
 echo "export https_proxy='$5'" >> /etc/profile.d/envvar.sh
-echo "export no_proxy=192.168.2.10,192.168.2.11,127.0.0.1,localhost,netmaster" >> /etc/profile.d/envvar.sh
-echo "export CLUSTER_NODE_IPS=192.168.2.10,192.168.2.11" >> /etc/profile.d/envvar.sh
+echo "export no_proxy=192.168.2.10,192.168.2.11,127.0.0.1,localhost,netmaster,192.168.2.12" >> /etc/profile.d/envvar.sh
```

Comment:
We should have based this on `node_ips` variable to begin with, instead of hard coding it here.
Do you mind changing it now?